docs/self-managed/operational-guides/update-guide/860-to-870.md

```diff
@@ -0,0 +1,14 @@
+---
+id: 860-to-870
+title: Update 8.6 to 8.7
+description: "Review which adjustments must be made to migrate from Camunda 8.6.x to Camunda 8.7.0."
+---
+
+The following sections explain which adjustments must be made to migrate from Camunda 8.6.x to 8.6.x for each component.
```

Comment:
Looks like this merged with a typo @entangled90. Please be sure to include `tech-writers` in your reviewers for most docs changes.